docs/index.asciidoc

```diff
@@ -1330,7 +1330,7 @@ WARNING: As of version `12.0.0` of this plugin, some configuration options have
 The plugin will fail to start if it contains any of these obsolete options.
 
 [cols="<,<",options="header",]
- |=======================================================================
+=======================================================================
```

Comment:
The pipe in Line 1333 needs to stay, please.

Reply:
🤦 i'm really doing poorly today. Need more coffee.